src/SFML/Window/VideoMode.cpp

```diff
@@ -76,6 +76,13 @@ bool VideoMode::isValid() const
     return std::find(modes.begin(), modes.end(), *this) != modes.end();
 }
 
+////////////////////////////////////////////////////////////
+Vector2u VideoMode::getAspectRatio() const
+{
+    const unsigned int divisor = std::gcd(this->size.x, this->size.y);
+
+    return Vector2u(this->size.x / divisor, this->size.y / divisor);
```

Comment:
```suggestion
    return size / divisor;
```
`sf::Vector2u` defines `Vector2u operator/(const Vector2u& left, unsigned int right);` so I believe this should be possible. Maybe we even collapse this whole function down to this:
```cpp
return size / std::gcd(size.x, size.y);
```